plugin/virtuemart/onepay/README.md

```diff
@@ -5,41 +5,130 @@
     <br>
     <h4>Compatibilidad</h4>
     <ul>
-      <li>Woocommerce 3.0.8 o inferior</li>
-      <li>Wordpress 4.8 or Inferior</li>
-      <li>PHP >= 5.6 and < 7.0</li>
+      <li>VirtueMart >= 3.x</li>
```

Comment:
Please don’t mix unrelated changes in the same PR. This should have been one PR for the onepay changes and one PR for the virtuemart updates